src/main/java/nextstep/ladder/domain/LadderName.java

```diff
@@ -0,0 +1,43 @@
+package nextstep.ladder.domain;
+
+import nextstep.ladder.exception.StringLengthException;
+
+import java.util.Objects;
+
+public class LadderName {
+    private static final int LETTER_LIMIT = 5;
+    private String laddername;
+
+    public LadderName(String ladderName) {
+        this.laddername = ladderName;
+    }
+
+    public static LadderName of(String ladderName) {
+        lengthValidation(ladderName);
+        return new LadderName(ladderName);
+    }
```

Comment:
```suggestion
    private LadderName(String ladderName) {
        this.laddername = ladderName;
    }

    public static LadderName of(String ladderName) {
        lengthValidation(ladderName);
        return new LadderName(ladderName);
    }
```
어차피 of가 똑같은 역할을 하고 있기 때문에 생성자는 private으로 지정해주세요!
무엇보다 생성자에 검증로직이 없기 때문에 of를 통해서만 인스턴스를 만들 수 있도록 해야합니다.